dev-utils/visualise-ui-hierarchy/tree2dot

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python3
+
+import sys
```

Comment:
Not too fond to introduce a new language in the repo... We would have to lint, and test, and do the whole nine yards. I understand this is a quick and dirty solution, but still. Also, quirks here: Enum needs 3.4, for instance. Would it be too much to ask to use any of the existing languages, from Node to Rust?

Reply:
Definitely not Rust, but I could redo it in JavaScript, yeah. In a second step if you don't mind :D